src/perl/lib/wtsi_clarity/epp/sm/pico_analyser.pm

```diff
@@ -6,7 +6,7 @@ use Readonly;
 use File::Temp qw/ tempdir /;
 use wtsi_clarity::file_parsing::dtx_concentration_calculator;
 use wtsi_clarity::file_parsing::dtx_parser;
-use wtsi_clarity::util::pdf::factory;
+use wtsi_clarity::util::pdf::factory::pico_analysis_results;
```

Comment:
Can you also add it to the dependencies, please
and there are some dependencies which are not being used, so please delete them.